### ISSUE TITLE
Tf Lite only support 4D l2_normalize?

### ISSUE THREAD

**Issue by apollo-time** [tensorflow/tensorflow#15620]
I build some feature extract network model and converted tflite using by toco successfully.
But I got error `"tensorflow/contrib/lite/kernels/l2norm.cc:47 NumDimensions(input) != 4 (2 != 4)`, when run interpreter->AllocateTensors().

I extract feature using by tf.nn.l2_normalize.
`embeddings = tf.nn.l2_normalize(prelogits, 1, 1e-10, name='embeddings')`
where prelogits is 2D tensor.
How can I extract normalized feature with tflite?

**Comment by asimshankar** [comment]
@aselle @andrehentz for thoughts here.

**Comment by tensorflowbutler** [comment]
Nagging Assignee: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.
